routes/employee_auth.js

```diff
@@ -167,14 +206,36 @@ const login = async (req, res) => {
         if (!(await bcryt.compare(req.body.password, existing_employee.employee_password_hash)))
             return res.json({ message: "Please review your credentials and try again!" })
 
-        const token = jwt.sign({ id: existing_employee.employee_id, admin: existing_employee.employee_is_admin }, config.access_token_secret);
-        console.log(token);
-        res.header("auth-token", token).send({ "token": token });
+        const access_token = create_access_token(existing_employee);
+        const refresh_token = jwt.sign({ id: existing_employee.employee_id, admin: existing_employee.employee_is_admin }, config.refresh_token_secret);
```

Comment:
What is the recommended expiry time on the refresh token?

Reply:
Because refresh token has to be stored on database, and removed from database when user log out. We do not need to set expiry time for refresh token

Reply:
Let's have a call tomorrow when you have time to discuss this a little more. I don't think I understand refresh tokens properly.